plugins/rebalance/rest.go

```diff
@@ -0,0 +1,231 @@
+package rebalance
+
+import (
+	//"context"
+	//"encoding/json"
+	"fmt"
+
+	"github.com/gluster/glusterd2/glusterd2/gdctx"
+	restutils "github.com/gluster/glusterd2/glusterd2/servers/rest/utils"
+	"github.com/gluster/glusterd2/glusterd2/store"
+	"github.com/gluster/glusterd2/glusterd2/transaction"
+	"github.com/gluster/glusterd2/glusterd2/volume"
+	"github.com/gluster/glusterd2/pkg/api"
+	"github.com/gorilla/mux"
+	"github.com/pborman/uuid"
+	log "github.com/sirupsen/logrus"
+	"net/http"
+)
+
+type Rebelstatus uint64
+
+const (
+	STARTED Rebelstatus = iota
```

Comment:
Try not to use all caps identifiers.

Reply:
Do you really want it to be `Rebelstatus` or `RebalStatus` :)